.gitignore

```diff
@@ -78,3 +78,4 @@ htmlcov/
 # System Files
 .DS_Store
 Thumbs.db
+shakecast/view/assets/dwr_logo2.png
```

Comment:
Instead of adding specific files here, use 

git update-index --assume-unchanged shakecast/view/assets/dwr_logo2.png

Reply:
Ok, thanks I am still new to to github and trying to figure out the best way to do things.